frontend/apps/core-app/src/components/RecipientListTable/RecipientListTableRow.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, HStack, Pressable, Text } from 'native-base';
+import { Cell, Row } from 'react-table';
+import React from 'react';
+
+import { RecipientListTableEntry } from './types';
+
+type Props<T extends Record<string, any>> = {
+  row: Row<T>;
+  onRowClick: (id: string) => void;
+};
+
+export const RecipientListTableRow: React.FC<
+  Props<RecipientListTableEntry>
+> = ({ row, onRowClick }) => {
+  return (
+    <Pressable
+      onPress={() => onRowClick(row.id)}
```

Comment:
Anonymous function. This one is particularly dangerous as this table could potentially render a lot of data, and this would cause a lot of rerendering.